include/lemlib/chassis/differential/drivetrain.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "pros/motor_group.hpp"
+#include "units/Pose.hpp"
+
+namespace lemlib::differential {
+/**
+ * @brief struct containing a left and right motor group representing a drivetrain
+ */
+struct Drivetrain {
+        pros::MotorGroup left; /** the left side of the drivetrain */
+        pros::MotorGroup right; /** the right side of the drivetrain */
+};
+
+/**
+ * @brief struct containing the position, velocity, and acceleration of a chassis
+ */
+struct ChassisState {
+        const units::Pose pose; /** pose of the chassis */
```

Comment:
dont these comments have to be on the line before?

Reply:
for the members? Pretty sure doxygen needs them to be on the same line

Reply:
clangd's intellisense does not work with them on the same line
![image](https://github.com/LemLib/LemLib/assets/80860310/c8ced363-c039-48f6-b4dc-20c416e48a6b)

Reply:
how annoying, fixed it